drafts/draft-20005.adoc

```diff
@@ -0,0 +1,224 @@
+blog_description: Open Liberty 20.0.0.5 allows developers to add a configurable attribute to an EJB Persistent Timer feature, setting a maximum amount of time allowing for the execution of a persistent timer to complete, before allowing another server to take over and run it instead.
+---
+= Persistent EJB Timers support failover between servers with Open Liberty 20.0.0.5
+
+Tom Jennings <https://github.com/tomjenningss>
+:imagesdir: /
+:url-prefix:
+:url-about: /about/
+
+// tag::intro[]
+Open Liberty 20.0.0.5 allows developers to add a configurable attribute to an EJB Persistent Timer feature, setting a maximum amount of time allowing for the execution of a persistent timer to complete, before allowing another server to take over and run it instead.
+
+In link:{url-about}[Open Liberty] 20.0.0.5:
+
+* <<EPT, EJB Persistent Timers>>
+* <<CLM, JAAS Custom Login Modules>>
+* <<OLL, Open Liberty Logging>>
+* <<JLO, Omit specified fields from JSON logging output>>
+
+View the list of fixed bugs in https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A20005+label%3A%22release+bug%22+[20.0.0.5].
+// end::intro[]
+
+If you're interested in what's coming soon in Open Liberty, take a look at our <<previews,current development builds>> which includes GraphQL with Open Liberty.
+
+// tag::run[]
+[#run]
+
+== Run your apps using 20.0.0.5
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.openliberty</groupId>
+    <artifactId>openliberty-runtime</artifactId>
+    <version>20.0.0.5</version>
+    <type>zip</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '[20.0.0.5,)'
+}
+----
+
+Or if you're using Docker:
+
+[source]
+----
+FROM open-liberty
+----
+//end::run[]
+
+Or take a look at our link:{url-prefix}/downloads/[Downloads page].
+
+[link=https://stackoverflow.com/tags/open-liberty]
+image::img/blog/blog_btn_stack.svg[Ask a question on Stack Overflow, align="center"]
+
+//tag::features[]
+
+[#EPT]
+=== Persistent EJB Timers coordination and failover across members
+
+Failover is a feature that adds a new configurable attribute, `missedTaskThreshold`, which specifies the maximum amount of time that you want to allow for an execution of a persistent timer to complete before allowing another server to take over and run it instead. Previously, it was only possible to partly coordinate automatic EJB persistent timers across multiple Open Liberty servers, by configuring the EJB timer service to persist timers to the same database. This caused a single timer instance to be created on one of the servers, without the ability to another server if the original server stops or crashes.
```

Comment:
this:

>Previously, it was only possible to partly coordinate automatic EJB persistent timers across multiple Open Liberty servers, by configuring the EJB timer service to persist timers to the same database. This caused a single timer instance to be created on one of the servers, without the ability to another server if the original server stops or crashes.

should be:

> Previously coordination of automatic EJB persistent timers across multiple Open Liberty servers was limited to ensuring that only a single instance of a timer executed across all server by configuring the EJB timer service on each to persist timers to the same database. If a server failed timers owned by that server would not run until it recovered, now they will be moved to run on other servers.

Reply:
@tomjenningss something here is still amiss, the current sentence has "to to" and doesn't read correctly
>This caused a single timer instance to be created on one of the servers but without the ability to to another server if the original server stops or crashes.